packages/flutter/test/widgets/listener_deprecated_test.dart

```diff
@@ -430,8 +438,8 @@ void main() {
       expect(bottomLeft.dy - topLeft.dy, scaleFactor * localHeight);
 
       final TestGesture gesture = await tester.createGesture(kind: PointerDeviceKind.mouse);
-      await gesture.addPointer();
       addTearDown(gesture.removePointer);
+      await gesture.addPointer(location: Offset.zero);
       await gesture.moveTo(topLeft - const Offset(1, 1));
```

Comment:
No, because sometimes we want to add a pointer without moving.
I mostly added the addPointer calls because that more closely represents what happens in a real interaction.

Reply:
I mean to merge this statement, not the entire API :)

Reply:
Oh, duh. :-) Yes, I'll fix that.